exonum-light-client/pom.xml

```diff
@@ -291,6 +305,12 @@
             <groupId>org.jacoco</groupId>
             <artifactId>jacoco-maven-plugin</artifactId>
             <version>0.8.3</version>
+            <configuration>
+              <excludes>
+                <!--Exclude auto generated lombok classes-->
+                <exclude>com/exonum/client/response/*</exclude>
```

Comment:
Is it the only thing we keep in this package? Why does it fail (and these classes have to be excluded), is there any possible fix?

I think it is fine to exclude them, but I'd add why it cannot be or shall not be fixed.

Reply:
There are a few enums in this package as well. If it's not ok - we can specify the exact list of classes.
It fails because `jacoco` is trying to check coverage for lombok generated files and can't find them at `src/main/java`.

Reply:
I see, so we exclude the shared package, but the comment says we exclude the auto-generated files (which we do, but also some other things). We can ignore some enums in the package, but if we don't specify the ignored one explicitly we might end up with more unreported classes (and never notice).

> Also, reason to check coverage for auto-generated code.

Lombok-processed classes does not entirely fall into such category (as _entirely_ generated classes). You might put as much code in lombok-annotated classes as you wish, so I don't think such comment is appropriate.

Reply:
_Normal_ generated classes are already excluded because they reside in `target/generated-sources` 🙃